pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -417,33 +417,44 @@ def test_constructor_with_categorical_categories(self):
     def test_from_codes(self):
 
         # too few categories
+        dtype = CategoricalDtype(categories=[1, 2])
         with pytest.raises(ValueError):
```

Comment:
Can you use the `match` parameter to check the error message here and other instances you're editing?  IIRC there's an effort underway to add these in across the tests.